cleanlab/internal/util.py

```diff
@@ -268,7 +268,41 @@ def round_preserving_row_totals(confident_joint) -> np.ndarray:
     ).astype(int)
 
 
-def estimate_pu_f1(s, prob_s_eq_1) -> float:
+
+def int2onehot(labels: List[List[int]]) -> NDArray:
+    """Convert list of lists to a onehot matrix for multi-labels
+
+    Parameters
+    ----------
+    labels: list of lists of integers
+      e.g. [[0,1], [3], [1,2,3], [1], [2]]
+      All integers from 0,1,...,K-1 must be represented."""
+
+    from sklearn.preprocessing import MultiLabelBinarizer
+
+    mlb = MultiLabelBinarizer()
+    return mlb.fit_transform(labels)
+
+# using generic NDArray,
+# if someone knows how to properly annotate 2D numpy arrays feel free.
+def onehot2int(onehot_matrix: NDArray) -> List[List[int]]:
+    """Convert a onehot matrix for multi-labels to a list of lists of ints
+
+    Parameters
+    ----------
+    onehot_matrix: 2D np.ndarray of 0s and 1s
+      A one hot encoded matrix representation of multi-labels.
+
+    Returns
+    -------
+    labels: list of lists of integers
+      e.g. [[0,1], [3], [1,2,3], [1], [2]]
+      All integers from 0,1,...,K-1 must be represented."""
+
+    return [list(np.where(row == 1)[0]) for row in onehot_matrix]
+
+
```

Comment:
These functions were moved to [cleanlab.internal.multilabel_utils.py](https://github.com/cleanlab/cleanlab/blob/f18279856fe5f00477462780a280f430d494faf3/cleanlab/internal/multilabel_utils.py#L72-L105), so you can remove these.

Re. your comment: 

- We need to put some care into how we'll annotate the numpy arrays, as the generic `np.ndarray` is too generic for the strict mypy check.
- `numpy.typing.NDArray` was introduced in version 1.21 of numpy, which Python 3.6 doesn't support. We're trying to maintain backwards compatibility with 3.6, so I don't think it would be worth it to use `NDArray` atm.
- The numpy arrays require two parameters (the shape and a data type) instead of one, so `np.ndarray[Any]` is not expected.